bundles/AdminBundle/Resources/public/js/pimcore/object/tags/booleanSelect.js

```diff
@@ -180,11 +180,17 @@ pimcore.object.tags.booleanSelect = Class.create(pimcore.object.tags.abstract, {
             options.labelWidth = this.fieldConfig.labelWidth;
         }
 
+        if (this.fieldConfig.labelAlign) {
+            options.labelAlign = this.fieldConfig.labelAlign;
+        }
+
         if (this.fieldConfig.width) {
             options.width = this.fieldConfig.width;
         }
 
-        options.width += options.labelWidth;
+        if (!this.fieldConfig.labelAlign || this.fieldConfig.labelAlign === 'left') {
+            options.width += options.labelWidth;
```

Comment:
what is the expectation here?
![image](https://user-images.githubusercontent.com/5137917/89277816-9a6acf00-d645-11ea-94d9-d0041ff7e8d0.png)

Reply:
I sum the width values with the calc css function now

Reply:
what about custom datatypes? I fear we have a BC break here.

Reply:
Can we cast it to an integer if it is numeric? Then everything stays the same in the javascript world.
Maybe in Service::enrichLayoutDefinition ?

Reply:
This is now only relevant for #6899, right? Yes for BC reason it will be good to cast values without unit to integer.

Reply:
Yes, exactly 👍